<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Photo.qrc">
    <normaloff>:/new/prefix1/photo/icon.png</normaloff>:/new/prefix1/photo/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/photo/login.png);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;border-radius:18px;padding:2px 4px;
background-image: url(:/new/prefix1/photo/white.png);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;border-radius:18px;padding:2px 4px;
background-image: url(:/new/prefix1/photo/white.png);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_log">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border:0px groove gray;border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border:0px groove gray;border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/photo/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/photo/min.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changepW">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border:0px groove gray;border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Photo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
